src/_gettsim/transfers/rente.py

```diff
@@ -787,19 +976,22 @@ def ges_rente_vorauss_besond_langj(ges_rente_wartezeit_45: float) -> bool:
     ges_rente_wartezeit_45
         See :func:`ges_rente_wartezeit_45`
 
+
     Returns
     -------
     Eligibility as bool.
 
     """
-    out = ges_rente_wartezeit_45 >= 45
 
-    return out
+    return ges_rente_wartezeit_45
 
 
 def ges_rente_wartezeit_5(
-    m_pflichtbeitrag: float, m_freiw_beitrag: float, m_ersatzzeit: float
-) -> float:
+    m_pflichtbeitrag: float,
+    m_freiw_beitrag: float,
+    m_ersatzzeit: float,
+    ges_rente_params: dict,
+) -> bool:
     """Aggregates time periods that are relevant for the general eligibility of the
```

Comment:
Can you please change the docstrings of all these function to reflect the changes in behavior (yes/no)? Maybe 

"Whether Allgemeine Wartezeit has been completed. Aggregates ..."

?

Reply:
done in commit [756e65b](https://github.com/iza-institute-of-labor-economics/gettsim/pull/634/commits/756e65b904b7e55545643ffa664ded50934f951a)